src/server/src/projects/entities/project.entity.ts

```diff
@@ -82,6 +82,9 @@ export class Project implements IProject {
   @Column({ type: "boolean", default: false })
   isFeatured: boolean;
 
+  @Column({ type: "numeric", default: 5 })
```

Comment:
A few issues here (and identical issues for `project-template.entity`)
 - If you look at the JSON response, it is returning `populationDeviation` as a string, not a number
 - If you look at the default value in the DB migration, it is setting it to the _string_ `'5'` not the number `5`.
  - We've _generally_ strived to use `camel_case` in the SQL schema by using the `name` option of `@Column({ ... })` to rename the columns (see `lockedDistricts` for an example)
  
Some small tweaks to the options will fix that
 - `"numeric"` is for arbitrary-precision numbers, `"double precision"` is what you want - that matches the underlying number type used by Javascript (https://www.postgresql.org/docs/13/datatype-numeric.html)
  - Using the lambda form of setting the default got it to make the default a number instead of a string cast to a number (which I guess would also work, but while you're updating this / reworking the migrations anyways you might as well change, it seems cleaner)
```suggestion
  @Column({ type: "double precision", name: "population_deviation", default: () => "5" })
```

Reply:
@maurizi Should this be:

```
  @Column({ type: "double precision", name: "population_deviation", default: () => DEFAULT_POPULATION_DEVIATION })
```

or does it need to be cast to a string first?

```
  @Column({ type: "double precision", name: "population_deviation", default: () => `${DEFAULT_POPULATION_DEVIATION}` })
```

Reply:
The latter is what we want, from looking at the SQL migrations generated after trying both options.